packages/main/src/Input.js

```diff
@@ -678,7 +677,10 @@ class Input extends UI5Element {
 
 	async _handleInput(event) {
 		await this.getInputDOMRef();
```

Comment:
On the 1st line we
`await this.getInputDOMRef();`
and then
`const inputDomRef = await this.getInputDOMRef();`

Remove the 1st line.